docs/pages/x/api/charts/area-element.json

```diff
@@ -15,6 +15,5 @@
     "name": "MuiAreaElement"
   },
   "filename": "/packages/x-charts/src/LineChart/AreaElement.tsx",
-  "demos": "<ul></ul>",
-  "packages": [{ "packageName": "@mui/x-charts", "componentName": "AreaElement" }]
```

Comment:
The `packages` field was used to build the imports but now we are building them differently, so it's just an unused field.